impl/client/src/main/javascript/web/pentaho/visual/config/vizApi.conf.js

```diff
@@ -781,7 +813,7 @@ define([
         }
       },
       {
-        priority: -2,
```

Comment:
We're actually giving a value of -5 for viz-author default styles, and -1 for apps, I think, so that would be even more appropriate.